Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -270,7 +283,7 @@ export const ProjectTree: React.FC<Props> = ({
       trigger: item.index,
       dialogName: dialog.id,
       isRoot: false,
-      projectId: currentProjectId,
```

Comment:
@beyackle I updated the link to use the projectId parameter as the currentProjectId seemed like the incorrect thing to pass.  Let me know if this wasn't a bug.

Reply:
No, that needed to be fixed, although it didn't matter in the single-bot case. I've got my own fix for this in the draft PR I have up, too, where we need to keep track of both a projectId and a skillId for each link.

Reply:
I also updated the order or params so that the first one was the primary (item, triggers, etc.) and then the later ones are the additional context (dialog, projectId)